Jenkinsfile

```diff
@@ -86,7 +86,7 @@ try {
 
                   stage('Static code analysis') {
                     try {
-                      gradle('realm', "findbugs pmd checkstyle ${abiFilter}")
```

Comment:
why PMD was disabled?

Reply:
It suddenly started reporting a ton of not-relevant errors. I didn't had time to figure out why or disable it. I added re-enabling PMD to the list of tasks in https://github.com/realm/realm-java/pull/6756